scripts/deb-install.sh

```diff
@@ -57,6 +57,7 @@ if grep -qi spider Packages; then
   spider_package_list=$(grep "^Package:" Packages |
     grep 'spider' | awk '{print $2}' | xargs)
 fi
+arch=$(deb_arch)
```

Comment:
notably this will correct:
```
+ [[ aarch64 != \a\m\d\6\4 ]]
+ [[ aarch64 != \a\r\m\6\4 ]]
+ bb_log_warn 'Due to MCOL-4123, Columnstore won'\''t be installed on aarch64'
+ set +x
WARNING: Due to MCOL-4123, Columnstore won't be installed on aarch64
```